test-utils/test-runner/src/client.rs

```diff
@@ -185,10 +184,16 @@ where
 
 	// Channel for the rpc handler to communicate with the authorship task.
 	let (command_sink, commands_stream) = mpsc::channel(10);
-
 	let rpc_sink = command_sink.clone();
 
-	let rpc_handlers = {
+	let rpc_builder = Box::new(move |_, _| {
+		let seal = ManualSeal::new(rpc_sink).into_rpc();
+		let mut module = RpcModule::new(());
```

Comment:
@maciejhirsz This is a bit fugly, the closure signature is `Box<dyn FnOnce(DenyUnsafe, SubscriptionTaskExecutor) -> Result<RpcModule<()>, Error>>,` so we have to build a dummy module and merge the two. Can we do better?

Reply:
This is why I originally had separate types for a module with context (with a generic) and plain module without any generic params :). We could add a method on `RpcModule<T>` that takes `self` by value and returns `RpcModule<()>`, which would achieve the same goal.

Reply:
> We could add a method on RpcModule<T> that takes self by value and returns RpcModule<()>, which would achieve the same goal.

https://github.com/paritytech/jsonrpsee/issues/558